api/src/dev/server/Dockerfile

```diff
@@ -1,28 +1,39 @@
-FROM alpine:3.7
+FROM ubuntu:18.04
```

Comment:
The changes in this file are the same as in #2471 except for `wait-for`/`wait-for-it`